test/test_transforms.py

```diff
@@ -1668,445 +2070,5 @@ def test_color_jitter():
-    def test_transformation_range(self, angle, translate, scale, shear, pil_image, input_img):
-        self._test_transformation(angle=angle, translate=(translate, translate), scale=scale,
-                                  shear=(shear, shear), pil_image=pil_image, input_img=input_img)
-
-
-def test_random_affine():
-
-    with pytest.raises(ValueError):
-        transforms.RandomAffine(-0.7)
-    with pytest.raises(ValueError):
-        transforms.RandomAffine([-0.7])
-    with pytest.raises(ValueError):
-        transforms.RandomAffine([-0.7, 0, 0.7])
-    with pytest.raises(TypeError):
-        transforms.RandomAffine([-90, 90], translate=2.0)
-    with pytest.raises(ValueError):
-        transforms.RandomAffine([-90, 90], translate=[-1.0, 1.0])
-    with pytest.raises(ValueError):
-        transforms.RandomAffine([-90, 90], translate=[-1.0, 0.0, 1.0])
-
-    with pytest.raises(ValueError):
-        transforms.RandomAffine([-90, 90], translate=[0.2, 0.2], scale=[0.0])
-    with pytest.raises(ValueError):
-        transforms.RandomAffine([-90, 90], translate=[0.2, 0.2], scale=[-1.0, 1.0])
-    with pytest.raises(ValueError):
-        transforms.RandomAffine([-90, 90], translate=[0.2, 0.2], scale=[0.5, -0.5])
-    with pytest.raises(ValueError):
-        transforms.RandomAffine([-90, 90], translate=[0.2, 0.2], scale=[0.5, 3.0, -0.5])
-
-    with pytest.raises(ValueError):
-        transforms.RandomAffine([-90, 90], translate=[0.2, 0.2], scale=[0.5, 0.5], shear=-7)
-    with pytest.raises(ValueError):
-        transforms.RandomAffine([-90, 90], translate=[0.2, 0.2], scale=[0.5, 0.5], shear=[-10])
-    with pytest.raises(ValueError):
-        transforms.RandomAffine([-90, 90], translate=[0.2, 0.2], scale=[0.5, 0.5], shear=[-10, 0, 10])
-    with pytest.raises(ValueError):
-        transforms.RandomAffine([-90, 90], translate=[0.2, 0.2], scale=[0.5, 0.5], shear=[-10, 0, 10, 0, 10])
-
-    # assert fill being either a Sequence or a Number
-    with pytest.raises(TypeError):
-        transforms.RandomAffine(0, fill={})
-
-    t = transforms.RandomAffine(0, fill=None)
-    assert t.fill == 0
-
-    x = np.zeros((100, 100, 3), dtype=np.uint8)
-    img = F.to_pil_image(x)
-
-    t = transforms.RandomAffine(10, translate=[0.5, 0.3], scale=[0.7, 1.3], shear=[-10, 10, 20, 40])
-    for _ in range(100):
-        angle, translations, scale, shear = t.get_params(t.degrees, t.translate, t.scale, t.shear,
-                                                         img_size=img.size)
-        assert -10 < angle < 10
-        assert -img.size[0] * 0.5 <= translations[0] <= img.size[0] * 0.5, ("{} vs {}"
-                                                                            .format(translations[0], img.size[0] * 0.5))
-        assert -img.size[1] * 0.5 <= translations[1] <= img.size[1] * 0.5, ("{} vs {}"
-                                                                            .format(translations[1], img.size[1] * 0.5))
```

Comment:
The assert msg here has been dropped for simplicity. See [#3952 comment](https://github.com/pytorch/vision/pull/3952#discussion_r645977808)